wikipedia.de/desktop/Banner.jsx

```diff
@@ -126,7 +126,7 @@ export default class Banner extends Component {
 							impressionCounts={ props.impressionCounts }
 							onFormInteraction={ this.onFormInteraction }
 							customAmountPlaceholder={ props.translations[ 'custom-amount-placeholder' ] }
-
+							onSubmit={ () => {} }
```

Comment:
shouldn't this be:
```suggestion
							onSubmit={props.onSubmit}
```

the onSubmit comes from the banner_presenter and seems to do the tracking, not sure, but should also be applied to WPDE?

Reply:
The DonationForm component posts the actual html form to spenden so I think it's okay. All the empty function does is block the error that's thrown right before the post happens.